src/main/java/org/italiangrid/storm/webdav/fs/Stat.java

```diff
@@ -0,0 +1,57 @@
+// SPDX-FileCopyrightText: 2014 Istituto Nazionale di Fisica Nucleare
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.italiangrid.storm.webdav.fs;
+
+import com.sun.jna.NativeLong;
+import com.sun.jna.Structure;
+import com.sun.jna.Structure.FieldOrder;
+
+@FieldOrder({
```

Comment:
we need a big warning here about the fact that this may be very much platform dependent, with a cross link to the check done at start time to protect about false assumptions